api/OffscreenCanvasRenderingContext2D.json

```
@@ -737,7 +737,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "105"
+              "version_added": false
```

Should be fixed in 116 per https://bugzilla.mozilla.org/show_bug.cgi?id=1792758

```suggestion
              "version_added": "116"
```